### PR TITLE
Fix Issue #1321 Preview Message button's accessibility issue

### PR DESCRIPTION
**Pull request by KavitaR45**
Added aria-label for the button <button class="preview-message-close-button">

### CODE REVIEW

File: package-lock.json
Comment (baptisteArno) [inline package-lock.json:1]: What's this?

File: packages/embeds/js/src/features/bubble/components/PreviewMessage.tsx [hunk 1]
```diff
@@ -67,6 +67,7 @@ const CloseButton = (props: {
 }) => (
   <button
     part="preview-message-close-button"
+    aria-label="preview-message-close-button"
```
Comment (baptisteArno) [inline packages/embeds/js/src/features/bubble/components/PreviewMessage.tsx:70]:
```suggestion
    aria-label="Close"
```

it should act like a real button label :)

**Review by baptisteArno — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (KavitaR45) [reply package-lock.json:1]: Ohh thanks for the suggestion, I thought close would be more common like other button would be using it. Thanks anyway.

**Review by baptisteArno — CHANGES_REQUESTED** [review]
Why is this PR adding a `package-lock.json` file?

**Pull request closed without merging**